nixos/modules/services/misc/dysnomia.nix

```diff
@@ -103,30 +103,36 @@ in
 
       properties = mkOption {
         description = "An attribute set in which each attribute represents a machine property. Optionally, these values can be shell substitutions.";
+        type = types.attrsOf types.str;
         default = {};
       };
 
       containers = mkOption {
         description = "An attribute set in which each key represents a container and each value an attribute set providing its configuration properties";
+        type = types.attrsOf types.attrs;
```

Comment:
Should this be more specific?

Reply:
I could go with:
types.attrsOf (types.attrsOf (types.either types.str types.int types.bool) 
what do you think?

Reply:
I think that would be more useful despite being complex.